stdlib/2and3/ssl.pyi

```diff
@@ -221,7 +222,7 @@ class SSLContext:
         def __init__(self, protocol: int) -> None: ...
     if sys.version_info < (3,) or sys.version_info >= (3, 4):
         def cert_store_stats(self) -> Dict[str, int]: ...
-    def load_cert_chain(self, certfile: str, keyfile: Optional[str] = ...,
+    def load_cert_chain(self, certfile: os.PathLike, keyfile: Optional[os.PathLike] = ...,
```

Comment:
This is problematic for two reasons: `os.PathLike` was only introduced in Python 3.6, so this will fail if type checking with lower version. This is also why Travis CI fails. Also, annotated like this, it will not accept `str`s anymore.

I recommend adding an alias like this somewhere at the top of the file and use it here:

```python
if sys.version_info < (3, 6):
    _Path = Text
else:
    _Path = Union[str, os.PathLike]
```

(Using `Text` instead of `str`, since in Python 2.7 using `unicode` as a path is most likely also fine.)